src/WWT.AppHost/Program.cs

```diff
@@ -17,13 +18,18 @@
     .WithHttpEndpoint(targetPort: 8080)
     .ExcludeFromManifest();
 
-builder.AddContainer("storage-explorer", "sebagomez/azurestorageexplorer")
-    .WithHttpEndpoint(targetPort: 8080)
-    .WithEnvironment(e =>
-    {
-        e.EnvironmentVariables["AZURE_STORAGE_CONNECTIONSTRING"] = new ConnectionStringReference(wwtFiles.Resource, optional: false);
-    })
-    .ExcludeFromManifest();
+// This container only supports x86_64
+var architecture = RuntimeInformation.OSArchitecture;
+if (architecture is Architecture.X64)
```

Comment:
nit: I tend to prefer not having a temp variable for things like this - it makes it a bit harder to read

```suggestion
if (RuntimeInformation.OSArchitecture is Architecture.X64)
```

Reply:
Yeah this is definitely better! I originally had something like `architecture != Architecture.Arm && architecture != Architecture.Arm64 && architecture != Architecture.Armv6` so the temporary was for readability.